infra/gp-multena/templates/system-datasources.yaml

```diff
@@ -1,13 +1,15 @@
 {{- if .Values.GrafanaOperatorDatasources }}
-apiVersion: integreatly.org/v1alpha1
-kind: GrafanaDataSource
+apiVersion: grafana.integreatly.org/v1beta1
+kind: GrafanaDatasource
```

Comment:
Da kommen wir uns in die quere, das habe ich schon in meiner PR upgedatet.

Reply:
dann mach noch einen merge?